mico-admin/src/app/app-detail/app-detail.component.html

```diff
@@ -16,3 +76,17 @@ <h3 class="ma3">Application {{application?.name}}</h3>
 </mat-card>
 
 <mat-card class="ma2">Endpoint: </mat-card>
+
+<mat-card class="ma2">
+    <mat-card-title>
+        Public IPs:
+    </mat-card-title>
+    <mat-card-content>
+        <div *ngFor="let ip of publicIps">
+            <a href="http://{{ip}}">
```

Comment:
add `target="_blank"` to tell the browser to open this link in a new tab. And `rel="noopener"` for better security.
<https://stackoverflow.com/questions/4964130/target-blank-vs-target-new/4964223#4964223>